Dockerfile

```diff
@@ -44,19 +44,19 @@ RUN UNAME_AT_COMPILE_TIME=true \
     PLACE_COMMIT=$PLACE_COMMIT \
     PLACE_VERSION=$PLACE_VERSION \
     crystal build \
-        --release \
-        --error-trace \
-        -o /app/rest-api \
-        /app/src/app.cr
+    --release \
+    --error-trace \
+    -o /app/rest-api \
+    /app/src/app.cr
 
 SHELL ["/bin/ash", "-eo", "pipefail", "-c"]
 
 # Extract binary dependencies
 RUN for binary in "/app/rest-api" "/bin/ping" "/bin/ping6"; do \
-        ldd "$binary" | \
-        tr -s '[:blank:]' '\n' | \
-        grep '^/' | \
-        xargs -I % sh -c 'mkdir -p $(dirname deps%); cp % deps%;'; \
+    ldd "$binary" | \
+    tr -s '[:blank:]' '\n' | \
+    grep '^/' | \
+    xargs -I % sh -c 'mkdir -p $(dirname deps%); cp % deps%;'; \
```

Comment:
Pls revert these changes 😢 🤷 🤔

Reply:
Or were these on purpose? My bad if so, I just indented them to be more block-like